app/assets/stylesheets/common/components/footer-nav.scss

```diff
@@ -78,6 +78,8 @@ body.footer-nav-visible {
 
 body.footer-nav-ipad {
   padding-top: var(--footer-nav-height);
+  box-sizing: border-box;
```

Comment:
This looks good to me. What do you think about going one step further and applying this to `body` at all times? I can see that being useful for themes that add padding to the body (and forget to check iPads, for example).

Reply:
Yes I thought of this and border-box all the thing is my religion :D But Im afraid this might break things?

Reply:
I would do it... only one way to find out if it will break things :-)